nevergrad/optimization/oneshot.py

```diff
@@ -89,42 +89,44 @@ class RandomSearchMaker(base.ParametrizedFamily):
         Provides a random recommendation instead of the best point so far (for baseline)
     middle_point: bool
         enforces that the first suggested point (ask) is zero.
-    quasi_opposite: str
+    opposition_mode: str or None
         symmetrizes exploration wrt the center: (e.g. https://ieeexplore.ieee.org/document/4424748)
              - full symmetry if "opposite"
              - random * symmetric if "quasi"
     cauchy: bool
         use a Cauchy distribution instead of Gaussian distribution
     scale: float or "random"
-        scalar for multiplying the suggested point values. If "random", this
-        used a randomized pattern for the scale.
+        scalar for multiplying the suggested point values, or string:
+         - "random": uses a randomized pattern for the scale.
+         - "auto": scales in function of dimension and budget (see XXX)
     recommendation_rule: str
-        "average_of_best" or "pessimistic"; "pessimistic" is the default and implies selecting the pessimistic best.        
+        "average_of_best" or "pessimistic"; "pessimistic" is the default and implies selecting the pessimistic best.
     """
 
     _optimizer_class = _RandomSearch
     one_shot = True
 
     # pylint: disable=unused-argument
     def __init__(self, *, middle_point: bool = False, stupid: bool = False,
-                 quasi_opposite: Optional[str] = None,
-                 cauchy: bool = False, scale: Union[float, str] = 1., 
+                 opposition_mode: Optional[str] = None,
+                 cauchy: bool = False, scale: Union[float, str] = 1.,
                  recommendation_rule: str = "pessimistic") -> None:
         # keep all parameters and set initialize superclass for print
-        assert quasi_opposite is None or quasi_opposite in ["quasi", "opposite"]
-        assert isinstance(scale, (int, float)) or scale == "random"
+        assert opposition_mode is None or opposition_mode in ["quasi", "opposite"]
+        assert isinstance(scale, (int, float)) or scale in ["auto", "random"]
```

Comment:
note that "super" is not tested anywhere, because it would have raised an exception